problems/removeOddNumbers.js

```diff
@@ -2,8 +2,13 @@
 * @param {number[]} arr - The input array
 * @returns {number[]} - The input array with all odd number removed
 */
-function removeOddNumbers() {
-
+function removeOddNumbers(arr) {
+let newArr = arr.filter(el => {
+    if(el % 2 === 0){
+        return arr[el]
```

Comment:
This might be working but isn't quite what you want. el is the element. That's like saying animals["cat"]. If animals is an array it doesn't make sense. Instead just return el % 2 === 0. This will return true for even numbers and false for odd. Filter will keep the even.